### PR TITLE
Add connecting to a running headless server.

### LINKED ISSUE #201: Add ability to specify server addr from command line
User should be able to connect to a headless Delve server using the terminal client.

### PR DESCRIPTION
**Pull request by tylerb**
Fix #201

Use `dlv connect "address"` to connect.

### CODE REVIEW

**Comment by tylerb** [comment]
Not sure why the linux test is failing here.

File: cmd/dlv/main.go [hunk 1]
```diff
@@ -222,9 +222,37 @@ starts and attaches to it, and enable you to immediately begin debugging your pr
 	}
 	rootCommand.AddCommand(attachCommand)
 
+	// 'connect' subcommand.
+	connectCommand := &cobra.Command{
+		Use:   "connect [pid]",
+		Short: "Connect to a headless debug server.",
+		Long:  "Connect to a headless debug server.",
+		Run: func(cmd *cobra.Command, args []string) {
+			addr := args[0]
```
Comment (derekparker) [inline cmd/dlv/main.go]: Should do a bounds check here, and print an error, like below.

Comment (tylerb) [inline cmd/dlv/main.go]: Fixed and pushed.

File: cmd/dlv/main.go [hunk 2]
```diff
@@ -222,9 +222,41 @@ starts and attaches to it, and enable you to immediately begin debugging your pr
 	}
 	rootCommand.AddCommand(attachCommand)
 
+	// 'connect' subcommand.
+	connectCommand := &cobra.Command{
+		Use:   "connect [pid]",
```
Comment (derekparker) [inline cmd/dlv/main.go]: Should be `[addr]` instead of pid.

**Comment by tylerb** [comment]
Whoops. That's what I get for going too fast.

**Comment by derekparker** [comment]
Merged, thanks!

**Pull request closed without merging**